cmd/gapir/cc/main.cpp

```diff
@@ -487,6 +487,10 @@ static int replayArchive(core::CrashHandler* crashHandler,
   std::unique_ptr<Context> context = Context::create(
       replayArchiveService, *crashHandler, resLoader.get(), &memoryManager);
 
+  if (replayArchiveService->getPayload("payload") == NULL) {
```

Comment:
`"payload"` string should probably be stored as a constant as it is used in different places in this file.